.gitignore

```diff
@@ -1,9 +1,22 @@
+# Environment files
+.env
 .env-back
 .env-front
+.envrc
 
+# citizenlab-ee files
 back/engines/ee
-node_modules
-
 citizenlab.config.ee.json
 
+
+# Cruft
 *.DS_Store
+
+docker-compose.override.yml
+
+# JetBrains settings files
+.idea/
+
+# VSCode settings files
```

Comment:
I'm not the biggest fan of adding editor-specific ignore files to the project, because they are specific to each developer (and the editor at use).

You could instead configure git locally to ignore certain files, it's 2 lines in your `.gitconfig`:

```
[core]
excludesfile = ~/.global_gitignore
```

And then add your editor specific files to that global gitignore. It will then also work for all git projects on your machine, not just this one here.

```
# Vim tag files
tags

# GNU Global tag files
GPATH
GRTAGS
GTAGS

# Org TODO
TODOs.org


# Spacemacs tag files
tags.lock
tags.temp

# Spacemacs temp files
.#*

# Emacs save files
.*~

# Vim save files
*.swp

# Generated by fast-mix plugin (oh-my-zsh)
.mix_tasks

# Generated by fast-rake plugin (oh-my-zsh)
.rake_tasks

# Directory local files (Emacs)
.dir-locals.el

# Mac OS
.DS_Store

# TypeScript Server
.log
```

Reply:
Agree and also think that we should remove the vscode-specific settings from the repo at some point. For now, though, the changes look ok 👍